openshift/templates/ecas-frontend/frontend.dc.yaml

```diff
@@ -0,0 +1,170 @@
+
+  apiVersion: template.openshift.io/v1
+  kind: Template
+  labels:
+    template: "${REPO_NAME}-template"
+  metadata:
+    name: "${REPO_NAME}-frontend-dc"
+    creationTimestamp:
+  objects:
+  - apiVersion: v1
+    kind: DeploymentConfig
+    metadata:
```

Comment:
Replace with Deployment object: https://eccbc.atlassian.net/wiki/spaces/EDKB/pages/536969221/DeploymentConfig+to+Deployments